### DISCUSSION TITLE
add unison in github-linguist

### DISCUSSION THREAD

**Discussion by ghost** [unisonweb/unison#3456] [Q&A] [answered]
Hi all. 

I would like to know if you've all thought about adding the unison language to github-linguist.

**Comment by ChrisPenner** [comment, marked as answer]
Hello 👋🏼 

Unison isn't quite like other languages in the sense that Unison stores all of its code in a binary format in a sqlite database, so Github is unfortunately not really a viable tool for programming in Unison.

Not to worry though, we're hard at work making [Unison Share](https://share.unison-lang.org/) a great collaboration tool (with syntax highlighting of course, and hyperlinking to boot!)

It'd be nice to have Unison in github linguist (and feel free to contribute it if you like) but it hasn't been a priority for us due to Unison Share 😄
